### DISCUSSION TITLE
'All items' link / button below article modules

### DISCUSSION THREAD

**Discussion by jjnxpct** [joomla/joomla-cms#29042] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

When using the category article module (or recent articles, or related articles) I always find myself manually adding a 'All items' or  'More items' link below the list of items to guide visitors to the page where the other items in that category are shown. For example: 

A recent news module has three recently added items. 
Below the three items there would be a link/ button to go to the blog/list page of all news items (most likely the 'news' menu item). 

This link can not be added in the module. Correct? So I need to add this in a template. Not a big deal for me because we always create our own templates. But how could a user do this without having to edit the template?

### Describe the solution you'd like

It might be a good idea to have an extra field in the mentioned modules to select a menu item you want a 'read all' link to point to. Or maybe just a text field where you could place the url yourself.

This would be a bit tricky when you have multiple categories selected in the recent items module.?

### Additional context
Maybe I am missing something ;-) It's also not a huge thing, but having a 'read all' or 'read more' button does seems logical to have. Or not?

**Comment by brianteeman** [comment]
> This would be a bit tricky when you have multiple categories selected in the recent items module.?

It would indeed. 

It would have to be a menu select and not a dynamic option
